src/components/IncidentsTable.tsx

```diff
@@ -12,7 +14,10 @@ type Props = {
 const IncidentsTableEmptyState: Component = () => {
   const navigate = useNavigate()
   const handleCreateIncident = () => {
-    navigate('/incidents/create')
+    // navigate('/incidents/create')
+    ;<Portal>
+      <CreateIncidentForm />
+    </Portal>
   }
```

Comment:
Hmm this doesn't look correct to me. Instead, triggering `handleCreateIncident` should open the modal.